_inc/client/components/navigation/index.jsx

```diff
@@ -18,6 +18,7 @@ import {
 	userCanManageModules as _userCanManageModules,
 	userCanViewStats as _userCanViewStats,
 } from 'state/initial-state';
+import { getSiteDevMode } from 'state/connection';
```

Comment:
Use `isDevMode`. This will allow you to avoid writing this
```
noRenderNavInDevMode = () => {
	return ! this.props.siteDevMode.isActive;
};
```
since you'll be able to add it to connect, instead of 
```
siteDevMode: getSiteDevMode( state ),
```
you'll add
```
isDevMode: isDevMode( state ),
```
and then in `render()` you can check for `this.props.isDevMode` instead of `this.noRenderNavInDevMode()`